index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="fr" xml:lang="fr" xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins:400,700,900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link href="./style.css" rel="stylesheet" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <h3 class="logo-text">My Logo</h3>
+      <img
+        class="menu"
+        src="./images/menu.svg"
+        alt="triangle with all three sides equal"
+        height="87"
+        width="100"
+      />
+    </header>
+    <main>
+      <h1 class="header-text">I'm Abraham Rodriguez Martinez to see you!</h1>
+      <p>
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <img
+        src="./images/Ellipse-3.svg"
+        class="e-3"
+        alt="bird"
+        height="10"
+        width="51"
+      />
+
+      <img
+        src="./images/Ellipse-11.svg"
+        class="e-11"
+        alt="bird"
+        height="10"
+        width="51"
+      />
+      <img
+        src="./images/Rectangle-4.svg"
+        class="r-4"
+        alt="bird"
+        height="38"
+        width="20"
+      />
+      <img
+        src="./images/Ellipse-5.svg"
+        class="e-5"
+        alt="bird"
+        height="11"
+        width="10"
+      />
+      <img
+        src="./images/Rectangle-7.svg"
+        class="r-7"
+        alt="bird"
+        height="11"
+        width="10"
```

Comment:
- [ ] This section has the proper background image (decorations should be with background images, not `img` tags). As Background image helps to write the text over them as well.